core/src/main/java/nu/yona/server/subscriptions/entities/User.java

```diff
@@ -25,6 +25,11 @@
 @Table(name = "USERS")
 public class User extends EntityWithID
 {
+	public enum Status
```

Comment:
If you change status to a boolean isConfirmed, the solution is to add this to the UserDTO.